### DISCUSSION TITLE
Masalah di kode geom_line

### DISCUSSION THREAD

**Discussion by AGelfP** [teamalgoritma/community#549] [Q&A] [answered]
Pertanyaan kali ini berhubungan dengan geom_line, saya berencana untuk membuat line plot, yang dimana x= tahun, y= views, dan line dipisahkan berdasarkan kategori. Gambarannya seperti ini:
![image](https://user-images.githubusercontent.com/122977451/219875958-79f5dbcf-3e23-4219-9f87-3adf9186c7db.png)

Namun pada saat saya jalankan di R, malah munculnya seperti ini:
![image](https://user-images.githubusercontent.com/122977451/219875977-ce71438d-11fd-4be4-8d67-d573eed62827.png)

Padahal saya sudah menyesuaikan kodenya berdasarkan apa yang telah diajarkan. Apakah masalahnya dalam tipe data. Terlampir adalah data yang digunakan:
![image](https://user-images.githubusercontent.com/122977451/219876027-8df42eb3-c75f-4693-98de-cb50830c6cd7.png)

Terima kasih lagi sebelumnya

**Comment by DeanAlexander27** [comment, marked as answer]
halo ka, izin membantu menjawab.. sepertinya tidak perlu facet_wrap . karena itu kan tujuannya memisahkan setiap category, ke masing" tabel nya

**Reply by AGelfP** [reply]
Nah kalau dihilangkan malah hasilnya gini Pak....
![image](https://user-images.githubusercontent.com/122977451/219933951-d9c29159-a7cb-463b-bfc7-02a2e219e839.png)

**Reply by DeanAlexander27** [reply]
boleh melihat tabel yt_trends nya seperti apa ya pak bentuknya ?

**Reply by AGelfP** [reply]
Ini kan Pak maksudnya?
![image](https://user-images.githubusercontent.com/122977451/219934191-e4d83bf8-78ba-476e-8d31-1d69fb3cbdc7.png)

**Reply by DeanAlexander27** [reply]
seperti ini pak ...
![image](https://user-images.githubusercontent.com/123348110/219934251-02e10297-a24c-4a58-bc8e-fa346565ecc0.png)

**Reply by AGelfP** [reply]
Terlampir ya...
![image](https://user-images.githubusercontent.com/122977451/219934563-5b5ac29f-904b-43bd-a501-002ff6ce6379.png)

**Reply by DeanAlexander27** [reply]
boleh di coba kode ini ya pak ..
geom_line(data = yt_trends, aes(x = year, y = views, color = category_id, group = category_id))

**Reply by AGelfP** [reply]
kalau dicoba pakai ggplot, hasilnya ga beda jauh sih
![image](https://user-images.githubusercontent.com/122977451/219935075-abd9afa0-1298-432e-b493-7f69adf03add.png)

Sedangkan kalau kode itu dijalankan langsung, malah ga muncul hasil
![image](https://user-images.githubusercontent.com/122977451/219935097-7c2711cf-bf53-41bd-b145-b6076a60c270.png)

**Reply by DeanAlexander27** [reply]
ah iya sorry , lupa sertain ggplot() nya 
boleh dicoba pak
ggplot() +
geom_line(data = yt_trends, aes(x = year, y = views, color = category_id, group = category_id))

**Reply by AGelfP** [reply]
![image](https://user-images.githubusercontent.com/122977451/219935227-9bd86c79-0af7-4da8-a593-74ba72a59be9.png)
sama hasilnya Pak wkwkwkwk

**Reply by DeanAlexander27** [reply]
wwwkwk, challenging juga ya pak... penasaran mau liat isi datanya seperti apa

**Reply by AGelfP** [reply]
file yang di folder materi utama IP Pak...youtubetrends.csv. Ini mau saya buat jadi LBB, takut ga keburu kalau nyari data baru wkwkwk.

Salah satu plot yang mau saya buat itu tren perkembangan category selama berjalannya waktu, dan gambarannya yang saya post di atas

**Reply by DeanAlexander27** [reply]
oke pak Gelfand,,, saya coba iseng buat ya pak... mungkin agak malaman...

**Reply by AGelfP** [reply]
![image](https://user-images.githubusercontent.com/122977451/219935662-0f369ce0-d2cf-4983-933e-031fbee208ba.png)
Ternyata masalah saya di wrangling Pak WKWKKWKWK. Sekarang udah muncul.

![image](https://user-images.githubusercontent.com/122977451/219935687-c0a5e8bc-1b80-4b5f-afc5-aaba3e88b81b.png)
wrangling saya seperti ini

**Reply by DeanAlexander27** [reply]
wkakak iya pak,, mantapppp.. dah aman berarti... sisa dashboardnya ya pak wkaka 👍 😆
